app/components/Button/connectButtonWrapper.tsx

```
@@ -87,113 +89,82 @@ export default function ConnectButtonWrapper() {
+          subtitle: 'The Stage is Yours',
+          title: 'Welcome to Creative TV',
+        },
+      }}
+      auth={{
+        getLoginPayload: async (params: GenerateLoginPayloadParams) => await generatePayload(params),
+        doLogin: async (
+          params: VerifyLoginPayloadParams,
+        ): Promise<void> => {
+          console.log('logging in...');
+          
+          const loginPayload: LoginPayload | void = await login(params);
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)